website/__tests__/redux/integration/ElementIntegrationTests.jsx

```diff
@@ -125,7 +135,9 @@ describe('PositionsIntegrationTests.', () => {
     const firebaseApp = initFirebase();
 
     // dispatch and persist the changes to firebase
-    testStore.dispatch(updatePositionAndPersist(elemId, updatedLoc)).then(() => {
+    testStore.dispatch(updateAndPersist(updatePosition, elemId, updatedLoc))
+        .then(() => {
+      /* TODO: Firebase should be mocked! Not a real call to Firebase! */
```

Comment:
we'll need to address this in another story. I think this is gonna be pretty involved or will require another package or something